.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,66 @@
+name: Java CI
+
+on: [pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    services:
+      postgres:
+        image: postgres
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: 'postgres'
+          POSTGRES_DB: test_db
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+
+    steps:
+    - name: Create test MySQL database
+      id: create_test_db
+      uses: mirromutth/mysql-action@v1.1
+      with:
+        host port: 3306
+        container port: 3306
+        mysql database: 'test_db'
+        mysql root password: 'root'
+
+    - uses: actions/checkout@v2.3.4
+
+    - name: Restore dependency cache
+      uses: actions/cache@v2.1.3
+      with:
+        path: ~/.m2/repository
+        key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+        restore-keys: |
+            ${{ runner.os }}-maven-
+
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+
+    - name: Configure connections to databases
+      id: configure_db_connections
+      run: cp extensions/database/tests/conf/github_actions_tests.xml extensions/database/tests/conf/tests.xml 
+
+    - name: Populate databases with test data
```

Comment:
Perhaps precede this with a "wait for databases to be available" check?

Reply:
Done when adding MySQL as a service.